EnglishFilter/sections/foreign.txt

```diff
@@ -738,6 +738,8 @@ tw.bid.yahoo.com###livePromote
 ||vali.*.cibntv.net^*.mp4?$mp4,domain=youku.com,important
 !+ PLATFORM(ext_chromium, ext_opera, ios, ext_safari, ext_android_cb)
 @@||vali.*.cibntv.net^*.mp4?$domain=youku.com,important
+! https://github.com/AdguardTeam/AdguardFilters/issues/28685
+tt1069.com##.wp .a_t > table[cellpadding="0"][cellspacing="1"]
```

Comment:
This space between `.wp` and `.a_t` is really needed?
I think that you can just use `.wp.a_t` in this case.

By the way, it seems that it's already fixed in `EasyList China`.